change-address/LETTERSPECS/BANNO.CHANGE.ADDR.V1.CFG

```diff
@@ -223,5 +226,47 @@ EB:Your financial institution address information has been updated. Please conta
 EB:you did not make these changes: (xxx) xxx-xxxx
 EB:Email line 5
 EB:Email line 6
+*
+* ALLOW ADDRESS INFORMATION CHANGES IN MEMO MODE
+*  Line must begin with "MMAIC:"
+*  Should the program allow address information changes when the system is in memo mode?
+*  If set to YES, memo mode allows address changes. A brief note is added to the 
+*  informational account note record being created and included in the member's email.
+*  If you set MMAIC to YES, and if SEND MEMBER CONFIRMATION EMAIL FROM parameter above "EF:" is also
+*  is filled in with a valid email the email will send from that address; otherwise
+*  the member's own email address is used. SENDMAIL functionality 
+*  must be turned on in your system.
+*  VALID VALUES: YES or NO
+*  DEFAULT:NO
+*
+MMAIC:
+*
+* MEMO MODE MEMBER CONFIRMATION EMAIL BODY
+*  This message will be emailed to the member when they complete address information changes,
+*  Symitar is in Memo Mode and the "ALLOW ADDRESS INFORMATION CHANGES IN MEMO MODE" (see above)
+*  parameter setting is set to "YES". Must set that parameter to "YES" above this section for it to work.
+*  Also, this section must be below the "" section.
+*  Line must start with "MMEB:". If a confirmation email is sent to the member,
+*  this text will be the body of the email sent to the member. You can either
+*  send the email in HTML format or in basic text.
+*  That is controlled but what you set above in parameter MEMBER CONFIRMATION EMAIL TYPE in "ET:" as HTML or TEXT.
```

Comment:
Sentence clarity - structure?

Reply:
Thanks @tkainz , I've cleaned that up. Then changes will be in the next commit.